### ISSUE TITLE
CMultiSelect - If user click on X to remove and CMultiSelect is inside <form />, page is submitted

### ISSUE THREAD

**Issue by matgenois** [coreui/coreui-vue#287]
Hi! And thank you for looking at my issue.
Has title mention, I have this weird issue where if the CMultiSelect is inside a <form />, the submit happens. I went in the html and it seems like the x button is missing type='button' since the default value of type is 'submit' : 
![image](https://github.com/coreui/coreui-vue/assets/7738711/7c596be4-3965-4326-991a-66419f81e601)
![image](https://github.com/coreui/coreui-vue/assets/7738711/96be0f12-3904-4a43-b118-3afbbe50174b)
If I add the type='button' manually on the button in the browser inspector, it works fine.
```
<template>          
  <CContainer fluid class="step-container py-2">
    <!-- no issue here -->
    <CFormMultiSelect id="customId"
        label="custom label"
        :options="getOptions()"
        v-model="customModel" />
    <form>
      <!-- issue here -->
      <CFormMultiSelect id="customId2"
        label="custom label 2"
        :options="getOptions()"
        v-model="customModel" />
```

**Comment by mrholek** [comment]
@matgenois which version of `@coreui/vue-pro` do you use?

**Comment by matgenois** [comment]
Bug is in 5.0.1 and 5.1.0

**Comment by mrholek** [comment]
@matgenois ok, I'm working on it.

**Comment by mrholek** [comment]
@matgenois fixed in v5.2.0

**Comment by matgenois** [comment]
Thank you!
